client/templates/Report/Report/_pdf_download_link.html.twig

```diff
@@ -2,6 +2,23 @@
 {% trans_default_domain translationDomain %}
 
 <a class="govuk-link button-link behat-link-download-pdf js-trackDownloadLink"
-   href="{{ path('report_pdf', {'reportId': report.id}) }}">
+   href="{{ path('report_pdf', {'reportId': report.id}) }}"
+   onclick="disableLink(this)">
     <i class="icon icon-pdf"></i>{{ 'downloadAsPdf' | trans() }}
 </a>
+
+<style>
+    .disabled {
+        pointer-events: none;
+        opacity: 0.5;
+        /* You can style the disabled link as per your requirement */
+    }
+</style>
+
+<script>
+    function disableLink(link) {
```

Comment:
Don't add any inline scripts, they'll be a problem when we fix the content security policy.

Reply:
Ah thanks! Will change this now!

Reply:
Wasn't a nice place to just put the function so had to follow the pattern for the others. Can you check and tick if that's ok? Added a test and checked manually and it works.